pkgs/development/go-modules/generic/default.nix

```diff
@@ -167,7 +167,7 @@ let
       if [ -z "$enableParallelBuilding" ]; then
           export NIX_BUILD_CORES=1
       fi
-      getGoDirs "" | xargs -n1 -P $NIX_BUILD_CORES bash -c 'buildGoDir install "$@"' --
```

Comment:
I think we still need -n1. However it would might be easier to just loop over `getGoDirs` in a bash for-loop.

Reply:
@Mic92 - I decided to keep the `xargs` instead of looping to avoid having to re-run the `go` compiler; if we're building a package with lots of Go modules, it avoids the startup time. I also want to keep the `xargs` command to ensure that we don't go past the maximum length of a command line for large packages.